src/layouts/core-dev-dot-layout/index.tsx

```diff
@@ -20,11 +20,15 @@ const CoreDevDotLayout = ({ children }: CoreDevDotLayoutProps) => {
 	const { asPath, pathname, isReady } = router
 
 	const isSwingset = asPath.startsWith('/swingset')
-	const isToastPath = pathname !== '/' && pathname !== '/_error' && !isSwingset
+	const isToastPath =
+		pathname !== '/' &&
+		pathname !== '/sign-up' &&
```

Comment:
Will refactor this to use config in a follow up task https://app.asana.com/0/1203652232454021/1204323512217789